modules/ob-enabling-builds-in-a-network-restricted-environment-using-the-buildah-build.adoc

```diff
@@ -0,0 +1,26 @@
+// Module included in the following assemblies:
+//
+// * builds/uninstalling-openshift-builds.adoc
+
+:_mod-docs-content-type: PROCEDURE
+[id='ob-creating-buildah-build-in-a-network-restricted-environment_{context}']
+= Creating buildah build in a network restricted environment
+
+You can create a `buildah` build in a network restricted environment by mirroring the images required by the `buildah` BuildStrategy. 
+
+.Prerequisites
+
+* Your cluster can connect and interact with the git source that you will use to create the buildah build.
+
+.Procedure
+
+. Run the following command to mirror the images required by the buildah buildStrategy:
++
+[source,terminal]
+----
+$ oc image mirror --insecure -a <registry-authentication> registry.redhat.io/ubi8/buildah@sha256:1c89cc3cab0ac0fc7387c1fe5e63443468219aab6fd531c8dad6d22fd999819e <mirror-registry>/<repo>/ubi8_buildah
+----
++
+
+. Perform the steps mentioned in the link:https://docs.redhat.com/en/documentation/builds_for_red_hat_openshift/1.1/html/work_with_builds/running-builds#ob-creating-a-buildah-build_using-builds[Creating a buildah build] section.
```

Comment:
Since the topic you're linking to is not external, i.e. it is in this Assembly, you cannot use a `link:` attribute. You could say "Perform the steps in the "Creating a buildah build" section." and if you want to get extra specific, put an `.Additional resources` section right after this step (in the Assembly) and an xref to that module. 
```suggestion
. Perform the steps mentioned in the "Creating a buildah build" section.
```